lib/tasks/import_licences.rake

```diff
@@ -0,0 +1,10 @@
+require "licence_importer"
+
+# Imports all published licences or you can filter by licence identifier stored
+# in `../common_licence_identifiers.rb`.
+
+desc "imports licences"
+task :import_licences, [:filter_by_licence_identifiers] => :environment do |_, args|
+  filter_by_licence_identifiers = args[:filter_by_licence_identifiers] || false
+  LicenceImporter.new(filter_by_licence_identifiers:).import
```

Comment:
is this missing the filter_by_licence_identifiers variable?
`LicenceImporter.new(filter_by_licence_identifiers: filter_by_licence_identifiers).import`

Reply:
No, the new Ruby styles expect you to use `(filter_by_licence_identifiers:)` rather than `(filter_by_licence_identifiers: filter_by_licence_identifiers)` if the key and value have the same name.

Reply:
Ah, not seen that previously (or not often enough to remember it - I will now). Thanks  Leena.